### PR TITLE
[zh] Sync grabage-collection.md

### PR DESCRIPTION
**Pull request by Skeletons430**

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
@Skeletons430 Thanks you for your contribution.
Could you please sign the CLA before the PR can be reviewed.
You can follow the steps documented here: [CLA Steps](https://git.k8s.io/community/CLA.md)

**Review by Arhell — COMMENTED** [review]
/easycla

**Comment by windsonsea** [comment]
Hi @Skeletons430 , this file has some more in-synced content with the en upstream #43544 , which you can also find by running this command on your `main` branch:

```
./scripts/lsync.sh content/zh-cn/docs/concepts/architecture/garbage-collection.md
```

**Comment by Skeletons430** [comment]
> Hi @Skeletons430 , this file has some more in-synced content with the en upstream #43544 , which you can also find by running this command on your `main` branch:
> 
> ```
> ./scripts/lsync.sh content/zh-cn/docs/concepts/architecture/garbage-collection.md
> ```
> 
> Would you like to include those differences into this PR?

Do not want to

**Comment by tengqm** [comment]
@Skeletons430 Please review https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#update
It is a hard requirement for keeping the whole page status under the radar. You can close the PR if you don't want to accept the reviewer's comment.

File: content/zh-cn/docs/concepts/architecture/garbage-collection.md [hunk 1]
```diff
@@ -273,6 +273,30 @@ until disk usage reaches the `LowThresholdPercent` value.
...
+regardless of disk usage. This is a kubelet setting that you configure for each node.
+
+To configure the setting, enable the `ImageMaximumGCAge`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) for the kubelet,
+and also set a value for the `ImageMaximumGCAge` field in the kubelet configuration file.
+
+The value is specified as a Kubernetes _duration_; for example, you can set the configuration
+field to `3d12h`, which means 3 days and 12 hours.
+-->
+#### 未使用的容器镜像的垃圾回收 {#image-maximum-age-gc}
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/architecture/garbage-collection.md:292]:
```suggestion
#### 垃圾收集未使用的容器镜像   {#image-maximum-age-gc}
```

File: content/zh-cn/docs/concepts/architecture/garbage-collection.md [hunk 2]
```diff
@@ -273,6 +273,30 @@ until disk usage reaches the `LowThresholdPercent` value.
...
+
+{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+
+作为一个alpha功能，你可以指定本地镜像无论磁盘使用情况如何，在多长时间内未被使用将进行垃圾回收。这是一个kubelet设置，你可以为每个节点进行配置。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/architecture/garbage-collection.md:296]:
```suggestion
作为一个 Alpha 特性，你可以指定本地镜像多长时间内未被使用将进行垃圾收集，与磁盘使用情况无关。
这是你可以为每个节点配置的一个 kubelet 设置。
```

File: content/zh-cn/docs/concepts/architecture/garbage-collection.md [hunk 3]
```diff
@@ -273,6 +273,30 @@ until disk usage reaches the `LowThresholdPercent` value.
...
+要配置此设置，请为kubelet启用`ImageMaximumGCAge`[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)，并在kubelet配置文件中设置`ImageMaximumGCAge`字段的值。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/architecture/garbage-collection.md:297]:
```suggestion

要配置此设置，请为 kubelet 启用`ImageMaximumGCAge`
[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)，
并在 kubelet 配置文件中设置 `ImageMaximumGCAge` 字段的值。
```

File: content/zh-cn/docs/concepts/architecture/garbage-collection.md [hunk 4]
```diff
@@ -273,6 +273,30 @@ until disk usage reaches the `LowThresholdPercent` value.
...
+该值被指定为Kubernetes持续时间，例如，你可以将配置字段设置为3d12h，表示3天12小时。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/architecture/garbage-collection.md:298]:
```suggestion

此值被指定为 Kubernetes **持续时间** ；例如，
你可以将配置字段设置为 3d12h，表示 3 天 12 小时。
```

**Review by windsonsea — COMMENTED** [review]
Thank you for adding the in-synced content.

Seems you need to read [中文本地化样式指南](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/) through. Also, [squash your commits](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits) into a single one.

/unhold

**Comment by windsonsea** [comment]
ping @Skeletons430

**Comment by windsonsea** [comment]
These changes have been merged in #44849 , so this will be closed.
/close

**Pull request closed without merging**